scripts/install-prometheus-operator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -o errexit
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+# Apply the crds and wait for them
+git clone https://github.com/prometheus-operator/kube-prometheus.git --depth 1 -b v0.11.0
+cd kube-prometheus   
+oc create -f manifests/setup
+oc get pods -A | grep prometheus
```

Comment:
probably you had the cycle in order to wait until the cluster comes up, in this case have some timeout, do not produce endless loops.

Reply:
I don't create here endless loops, maybe I missing something.
can you tell me where have endless loops? @rdavid

Reply:
there was the loop yesterday with `until`.